test/loader/adapter_registry/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: MIT
+
+set(DUMMY_LIB_DIR ${CMAKE_BINARY_DIR}/bin/${CMAKE_BUILD_TYPE}/adapter-reg)
+
+if(MSVC)
+    set(DUMMY_LIB_DIR ${CMAKE_BINARY_DIR}/bin/${CMAKE_BUILD_TYPE}/adapter-reg)
```

Comment:
Please, check if both MSVC and non-MSVC paths could be resolved with `CMAKE_CURRENT_BINARY_DIR`:
```
${CMAKE_CURRENT_BINARY_DIR}/adapter-reg
```

Reply:
It's no longer relevant.